apps/common/lib/lexical/ast/analysis.ex

```diff
@@ -365,6 +367,40 @@ defmodule Lexical.Ast.Analysis do
     State.push_import(state, Import.new(state.document, quoted, module))
   end
 
+  # require of a module using as
```

Comment:
IMO examples like this are a lot more clear than writing out in English

```suggestion
  # require MyModule, as: Alias
```